search.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // SearchResult - Occurrence of an emoji in a string
 type SearchResult struct {
-	Match       interface{}
```

Comment:
This should have never been an interface, it leads to unnecessary type conversion/checking.